javascriptv3/example_code/cross-services/lambda-for-browser/README.md

```diff
@@ -16,318 +17,123 @@ This tutorial shows you how to use the AWS SDK for JavaScript V3 API to invoke t
-* Running this code might result in charges to your AWS account. We recommend you destroy the resources when you are finished. For instructions, see [Destroying the resources](#destroying-the-resources).
-* This tutorial is written to work with the specific versions defined in the *package.json*. If you change these versions, the tutorial may not work correctly.
+
+- We recommend that you grant this code least privilege, or at most the minimum permissions required to perform the task. For more information, see [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) in the _AWS Identity and Access Management User Guide_.
+- This code has not been tested in all AWS Regions. Some AWS services are available only in specific [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+- Running this code might result in charges to your AWS account. We recommend you destroy the resources when you are finished.
 
 ## Create the resources
-You can create the AWS resources required for this cross-service example the [AWS Management Console](#create-the-resources-using-the-aws-management-console)
-
-#### Create an unauthenticated user role
-1. Open [AWS Cognito in the AWS Management Console](https://aws.amazon.com/cloudformation/), and open the *Stacks* page.
-2. Choose **Manage Identity Pools**.
-3. Choose **Create new identity pool**.
-4. In the **Identity pool name** field, give your identity pool a name.
-5. Select the **Enable access to unauthenticated identities** checkbox.
-6. Choose **Create Pool**.
-7. Choose **Allow**.
-8. Take note of the **Identity pool ID**, which is highlighted in red in the **Get AWS Credentials** section, and choose **Edit identity pool**. 
-9. Take note of the name of the role in the **Unauthenticated role** field.
-
-#### Adding permissions to an unauthenticated user role
-1. Open [IAM in the AWS Management Console](https://aws.amazon.com/iam/), and open the *Roles* page.
-2. Search for the unauthenticated role you just created.
-3. Open the role.
-4. Click the down arrow beside the policy name.
-5. Choose **Edit Policy**.
-6. Choose the **JSON** tab.
-7. Delete the existing content, and paste the code below into it.
+
+**Note:** The links to the console in the following sections will take a best guess at your intended region. Make sure to create the resources in the same region.
+
+### Create an identity pool and guest role
+
+1. Open [AWS Cognito in the AWS Management Console](https://console.aws.amazon.com/cognito/).
+1. Select `Identity pools` in the sidebar.
+1. Click `Create identity pool`.
+1. Check the `Guest access` box.
+1. Click `Next`.
+1. Select `Create a new IAM role`.
+1. Enter a role name in the `IAM role` field.
+1. Click `Next`.
+1. Enter a name for the identity pool.
+1. Click `Next`.
+1. Review the information and click `Create identity pool`.
```

Comment:
```suggestion
1. Choose `Create identity pool`.
1. Select the `Guest access` check box.
1. Choose `Next`.
1. Select `Create a new IAM role`.
1. Enter a role name in the `IAM role` field.
1. Choose `Next`.
1. Enter a name for the identity pool.
1. Choose `Next`.
1. Review the information and choose `Create identity pool`.
```

Reply:
@lkdavies is `Choose` right for a button? To me, choose sounds like I have a list of options.